### ISSUE TITLE
Need to "Reapply All Breakpoints"

### ISSUE THREAD

**Issue by gludion** [Microsoft/vscode-chrome-debug-core#305]
Hello,
I've struggled to debug typescript properly in vscode because all breakpoint were "broken" (message: "Breakpoint ignored because generated code not found (source map problem?").
I've tried the techniques [described here ](https://github.com/Microsoft/vscode-chrome-debug#sourcemaps) to log those issues. 

Then I realized that launch.json needed a more precise "webRoot":
"webRoot": "${workspaceFolder}/my/Adequate/subFolder/Path"
Then the output of ".scripts" was correct path for source files, but the breakpoint were still broken, and I still had those messages in log file:
`
To client: {"seq":0,"type":"response","request_seq":3,"command":"setBreakpoints","success":true,"body":{"breakpoints":[{"verified":false,"line":217,"message":"Breakpoint ignored because generated code not found (source map problem?).","id":1000},{"verified":false,"line":220,"message":"Breakpoint ignored because generated code not found (source map problem?).","id":1001}]}}
`
After some hours I tried to add a breakPoint during a debug sesssion and -miracle- all previous breakpoint were fixed! (but NOT the newly added breakpoint, lol?). 
I found a tip in the VScode documentation https://code.visualstudio.com/docs/editor/debugging:
> The Reapply All Breakpoints command sets all breakpoints again to their original location. This is helpful if your debug environment is "lazy" and "misplaces" breakpoints in source code that has not yet been executed.
So I tried launching my chrome debug session, all breakpoints broken as usual, then right-clicked "Reapply all Breakpoints" and all was fixed! 

What? So many hours lost, while my setup was correct...
What is this "lazy" environment? vscode? Chrome? What's the point of the whole setup if the provided feedback is wrong?
Questions: 

- is  there a way to force this "environment" to provide correct feedback, that is not say that some path can be resolved is it can be resolved later without changing anything.

- is there a way to "reapply all breakpoints" automatically?

**Comment by roblourens** [comment]
This is could be https://github.com/Microsoft/vscode/issues/45657#issuecomment-373556464, please update the Chrome extension and let me know if that fixes it.

**Comment by gludion** [comment]
yes, reading the conclusion of your link, I confirm that I noticed that strange drive lower/uppercase change in log file (and also the various inconsistent ways to separate folder names: "/" or "\" or even "\\", )
My first thought was "wow, hopefully those file path naming cannot possibly be an issue in any modern OS or software"... Well...

ps: my vscode extension "Debugger for Chrome" is up to date.

**Comment by roblourens** [comment]
Are you saying that there is still a problem when using version 4.2.1/

**Comment by gludion** [comment]
I didn't notice 4.2.1. vscode suggests only 4.1.0. That's why I suppose it was up to date.
Tried uninstall + re-install, but it's still 4.1.0.
Also on official page: https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome (shows 4.1.0).

**Comment by roblourens** [comment]
This is the number that matters: 
![image](https://user-images.githubusercontent.com/323878/37537564-97173352-2913-11e8-87fe-f32b34ff0afc.png)

The other is just because I forgot to push tags since 4.1.0

**Comment by gludion** [comment]
yes it now works perfectly well with 4.2.1
Thank you very much!
